tests/bitcoin/BitcoinClient.spec.ts

```diff
@@ -392,7 +393,8 @@ describe('BitcoinClient', async () => {
       };
 
       spyOn(BitcoinClient as any, 'createBitcoreTransactionWrapper').and.returnValue(transaction);
-      const spy = mockRpcCall('getblock', [hash, 2], blockData);
+      const expectedIsWalletRpc = false;
```

Comment:
You can pull the scope of `expectedIsWalletRpc` to the describe as all tests define this to the same value.

Reply:
Pulled all `expectedIsWalletRpc` into describe when there are more than two references of the variable, otherwise left them in place (majority of the cases).